packages/cli/src/commands/generate/scaffold/templates/components/NameForm.tsx.template

```diff
@@ -27,21 +37,39 @@ const ${singularPascalName}Form = (props) => {
           titleClassName="rw-form-error-title"
           listClassName="rw-form-error-list"
         />
-<% editableColumns.forEach(column => { %>
+      <% editableColumns.forEach(column => { %>
         <Label
           name="${column.name}"
           className="rw-label"
           errorClassName="rw-label rw-label-error"
         >
           ${column.label}
         </Label>
-        <${column.component}
-          name="${column.name}"
-          ${column.defaultProp}={${column.deserializeFunction && column.deserializeFunction + '('}props.${singularCamelName}?.<%= column.name %>${column.deserializeFunction && ')'}}
-          className="rw-input"
-          errorClassName="rw-input rw-input-error"<%= !column.validation ? '' : `
-          validation=${column.validation}` %>
-        />
+
+        <% if (column.isEnum) { %>
```

Comment:
It'd be nice if we could make this whole thing flexible/generic enough to not have to make a special case for enums. Like maybe always have a loop here, but enums would be the only one (right now) that would loop more than once. Something like that.

Reply:
True. But maybe that can be an optimization. There is some added logic for a single enum None value, too, now.

Reply:
👍